### ISSUE TITLE
Compiled binary for localnet 

### ISSUE THREAD

**Issue by mfornet** [near/nearup#31]
Currently localnet doesn't support using compiled binary for the platform, but will require `binary-path`. Reuse the same logic implemented in `nodelib.py` to download compiled binary if it exists and `binary-path` was not provided.

**Comment by ailisp** [comment]
Which binary do we want to use as default? devnet (latest) or testnet (latest stable)

**Comment by mfornet** [comment]
I have no particular preference, but probably `devnet` makes more sense as it should be the goto network to develop on, which I presume will be the main use case of this feature.

**Comment by mikroskeem** [comment]
I would prefer testnet `neard` as default, but adding additional flag for picking the `neard` flavour would be useful in the future for those who actually need it.

**Comment by mikroskeem** [comment]
To get testnet up and running easily (in Docker) currently, you can do this however:

docker-compose.yml
```yaml
version: '2.4'

services:
  nearup:
    #image: nearprotocol/nearup:latest
    build: .
    command: 'run localnet --binary-path /neard-local'
    init: true
    volumes:
      - data:/root/.near
    ports:
      - 127.0.0.1:3030:3030

volumes:
  data:
```

Dockerfile:
```dockerfile
FROM nearprotocol/nearup:latest

COPY --from=nearprotocol/nearcore:latest /usr/local/bin/neard /neard-local/neard
```

Note that having `init: true` (or Docker CLI equivalent `--init`) is required, see [this article](https://blog.phusion.nl/2015/01/20/docker-and-the-pid-1-zombie-reaping-problem/) why.

**Comment by chefsale** [comment]
I think this was done intentionally, as different networks use different binaries. We have 3 binaries:

 - betanet - nighly release
 - testnet - perf release (stable with perf logging enabled)
 - mainnet - stable release.

In localnet it is unclear which of these you want to run. We could add a separate flags to specify which one to download. If this is a blocker we can prioritize implementing this, but initial implementation for this was for local dev setup where you specify the binary you want to run.

**Comment by chefsale** [comment]
@mfornet created a PR to just download testnet binary for now: https://github.com/near/nearup/pull/174

**Comment by chefsale** [comment]
Landed and created a pypi 0.9.1 release.
